source/installation/kafka-deployments/migrator-with-kafka.txt

```diff
@@ -16,41 +16,35 @@ Integrating Relational Migrator with Kafka
    :depth: 1
    :class: singlecol
 
-Relational Migrator uses `Apache Kafka 
+{+rel-mig+} uses `Apache Kafka 
 <https://www.mongodb.com/docs/kafka-connector/current/introduction/kafka-connect/#apache-kafka>`__ 
 as a robust transport layer to migrate data from a source relational database 
-to MongoDB. Relational Migrator can be used as a Kafka Connect plugin that 
+to MongoDB. {+rel-mig+} can be used as a Kafka Connect plugin that 
 improves resilience and scalability of big data migration jobs.
 
 Use Cases
 ---------
 
-Relational Migrator with Kafka is intended for large and long running jobs.
+{+rel-mig+} with Kafka is intended for large and long running jobs.
 Kafka's built-in resilience ensures the migration job can recover and continue 
 running if any component becomes temporarily unavailable. 
 
 Behavior
 --------
 
-Relational Migrator works with Kafka as a Kafka Connect sink connector. A 
-`Debezium connector <https://debezium.io/documentation/reference/stable/connectors/index.html>`__ 
-captures data events from your source database, while Relational Migrator 
-interprets and transforms events from the Debezium source and sends them to a 
-MongoDB cluster that acts as a data sink.
-
-Architecturally, Relational Migrator is deployed within Kafka as a Kafka 
+Architecturally, {+rel-mig+} is deployed within Kafka as a Kafka 
```

Comment:
(consistency)
Below, in an unedited part. You might consider: ... change data capture (CDC). This will keep consistent with other pages introducing CDC.